### PR TITLE
RFR: Update begets

### PR DESCRIPTION
**Pull request by digitronik**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,19 +2,24 @@
...
+.. image:: https://codecov.io/gh/RedHatQE/widgetastic.core/branch/master/graph/badge.svg
+    :target: https://codecov.io/gh/RedHatQE/widgetastic.core
 
 .. image:: https://readthedocs.org/projects/widgetastic/badge/?version=latest
     :target: http://widgetastic.readthedocs.io/en/latest/?badge=latest
     :alt: Documentation Status
 
-.. image:: https://www.quantifiedcode.com/api/v1/project/2f1c121257cc44acb1241aa640c4d266/badge.svg
-  :target: https://www.quantifiedcode.com/app/project/2f1c121257cc44acb1241aa640c4d266
-  :alt: Code issues
```
Comment (digitronik) [inline README.rst:17]: dropping code issues

File: README.rst [hunk 2]
```diff
@@ -26,8 +31,11 @@ Licensed under Apache license, Version 2.0
 *WARNING:* Until this library reaches v1.0, the interfaces may change!
 
 Projects using widgetastic
---------------------------
-- ManageIQ integration_tests: https://github.com/ManageIQ/integration_tests
+---------------------------
+- ManageIQ [`integration_tests <https://github.com/ManageIQ/integration_tests>`_]
+- Satellite [`airgun <https://github.com/SatelliteQE/airgun>`_]
+- Cloud Services [`insights-qe <https://gitlab.cee.redhat.com/insights-qe>`_]
```
Comment (quarckster) [inline README.rst:37]: I don't think putting an internal url makes sense here.

**Review by quarckster — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (digitronik) [reply README.rst:37]: sure restoring old style.

Comment (digitronik) [reply README.rst]: @quarckster done

Comment (quarckster) [reply README.rst]: @digitronik The issue is not in the style. This url https://gitlab.cee.redhat.com/insights-qe is not public and it should be removed.

Comment (digitronik) [reply README.rst]: ahhhh sure thing

**Review by quarckster — APPROVED** [review]

**Pull request merged**